hal/renesas-rx.c

```diff
@@ -433,7 +433,14 @@ void hal_init(void)
 
 void hal_prepare_boot(void)
 {
-
+    /* make sure interrupts are disabled */
```

Comment:
This is usually done in `do_boot` because it's in general architecture-specific rather than target-. Check e.g. src/boot_arm.c:357.
Consider moving to `boot_renesas.c`.